.gitignore

```diff
@@ -0,0 +1,66 @@
+# Python build artifacts
```

Comment:
You don't need this file at all. Meson doesn't let you build the project inside a source tree, it demands separated build directory.

Reply:
Removed